### ISSUE TITLE
fix tutorials that are not run in CI

### ISSUE THREAD

**Issue by clee2000** [pytorch/tutorials#2033]
Here is a list of all the tutorials that are known to run in 0.000 sec, indicating that they probably didn't run any python code in them:
https://github.com/pytorch/tutorials/blob/master/.jenkins/validate_tutorials_built.py#L15

We would like to either fix the tutorial, or move the tutorial to the `ok_to_not_run` list with a comment about why it's ok that the tutorial ran in 0 seconds.  For example, `"beginner/basics/intro.html",  # no code`.

**Comment by svekars** [comment]
A quick question here - in conf.py, we specify a pattern for sphinx_gallery to run the tutorials that end with `_tutorial.py`. I believe a lot of the tutorials in the list of tutorials that don't run don't have that suffix. So it's by design, I think?

**Comment by clee2000** [comment]
yes, but it was changed in https://github.com/pytorch/tutorials/pull/2034

I'm not sure if these are actually supposed to not be run or if they just happened to be named without `_tutorial.py` but still should be run, and based on what was said in the group chat, it seemed like they should be run
